test/cql-pytest/test_bloom_filter.py

```diff
@@ -29,3 +30,19 @@ def test_bloom_filter(scylla_only, cql, test_keyspace, N, M, fp_chance):
         ratio = fp / M
         assert ratio >= fp_chance * 0.7 and ratio <= fp_chance * 1.15
             
+# Test very small bloom_filter_fp_chance settings.
+# The Cassandra documentation suggests that bloom_filter_fp_chance can be set
+# to anything between 0 and 1, and the Datastax documentation even goes further
+# and explains that 0 means "the largest possible Bloom filter".
+# But in practice, there is a minimal false-positive chance that the Bloom
+# filter can possibly achieve and Cassandra refuses lower settings (see
+# CASSANDRA-11920) and Scylla should do the same instead of crashing much
+# later during a memtable flush as it did in issue #11524.
+@pytest.mark.parametrize("fp_chance", [1e-5, 0])
```

Comment:
Perhaps the test should include a case where is `bloom_filter_fp_chance > 1.0`, too? Although the main purpose of this PR is fixing the case where the bloom filter chance is too low, it also adds a new validation check for the case where it is too high.

Reply:
Good idea. I'll produce a v2.